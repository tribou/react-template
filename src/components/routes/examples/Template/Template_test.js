// @flow

// Enzyme docs:
// http://airbnb.io/enzyme/docs/api/index.html

import React from "react";
import { shallow } from "enzyme";
import Template from "./Template";

const mockProps = {
  url: "http://www.example.com"
};

it("<Template> renders the Template content", () => {
  const wrapper = shallow(<Template {...mockProps} />);

  expect(wrapper).toMatchSnapshot();
});
