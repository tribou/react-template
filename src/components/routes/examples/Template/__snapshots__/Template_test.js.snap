// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Template> renders the Template content 1`] = `
<div
  className="template"
>
  http://www.example.com
</div>
`;
